Gems/Atom/RHI/Code/Include/Atom/RHI/MultiDeviceResource.h

```diff
@@ -84,4 +84,27 @@ namespace AZ::RHI
         //! The version is monotonically incremented any time the backing resource is changed.
         uint32_t m_version = 0;
     };
+
+    class MultiDeviceBuffer;
+    class MultiDeviceImage;
+    class ResourceView;
+
+    /**
```

Comment:
- Use //!
- The comments is a copy paste and it is not clear why this class is needed